src/tests/src/libs/antares/study/thermal-price-definition/thermal-price-definition.cpp

```diff
@@ -174,104 +166,73 @@ BOOST_AUTO_TEST_CASE(EconomicInputData_loadFromFolder_failing_not_enough_value)
 
 BOOST_AUTO_TEST_CASE(EconomicInputData_loadFromFolder_working_with_too_much_value)
 {
-    createIniFile();
-    createFuelCostFile(10000);
+    CO2CostFile f(10000);
 
     clusterList.loadFromFolder(*study, folder, area);
 
     EconomicInputData eco;
     BOOST_CHECK(eco.loadFromFolder(*study, folder));
 }
 
-BOOST_AUTO_TEST_CASE(checkFuelAndCo2_basic_working)
+BOOST_AUTO_TEST_CASE(checkCo2_basic_working)
```

Comment:
For this test and the next 2, the fixture is triggered but unused.
Maybe we should have a different fixture for these 3 tests.

Reply:
done